packages/modeling/src/operations/booleans/subtractGeom2.js

```diff
@@ -1,17 +1,13 @@
-import { flatten } from '../../utils/flatten.js'
-
 import { DIFFERENCE } from './martinez/operation.js'
 import { boolean } from './martinez/index.js'
 
 /*
  * Return a new 2D geometry representing space in the first geometry but
  * not in the subsequent geometries. None of the given geometries are modified.
- * @param {...geom2} geometries - list of geometries
+ * @param {...Geom2} geometries - a flat list of 2D geometries
```

Comment:
Geom2[]

Reply:
I am happy to make these changes, but I am curious on your thoughts as to why?

"..." is legal in jsdoc as a way to define something as an array type. So `...Geom2` and `Geom2[]` mean the same thing. I fixed the capitalization of the types in this PR, but I just left it as "..." for consistency. Should we make a lint rule that prefers [] over ...?

Reply:
Hmm... I suggested the change because the API is really expecting an array now, rather than an arbitrary number of parameters. Does that make sense?

I don't really have a preference on the capitalization, etc in the documentation but jsdoc needs to spit out something nice for the users.

I suspect that there will be a few rounds of documentation changes for V3.